packages/protocol/test-sol/unit/common/CeloDistributionSchedule.t.sol

```diff
@@ -69,14 +69,13 @@ contract CeloDistributionScheduleTest is Test, Constants, IsL2Check {
     setUpL1();
 
     // Setup L2 after minting L1 supply.
-    registryAddress = proxyAdminAddress;
-    deployCodeTo("Registry.sol", abi.encode(false), registryAddress);
-    registry = IRegistry(registryAddress);
```

Comment:
Todo: Check if I use constants correctly.

I'm a little confused about why a registry contract is made using the `proxyAdminAddress`. It might be a shorter way of both activating the L2 (by deploying arbitrary code at the `proxyAdminAddress` _and_ having a registry. If that's the case, it's also a little confusing to read and make sense of.

```
registryAddress = proxyAdminAddress;
deployCodeTo("Registry.sol", abi.encode(false), registryAddress);
registry = IRegistry(registryAddress);

// ...

celoToken.setRegistry(registryAddress);
```

Reply:
For now, I solved this by keeping the intent of the original code as is 

https://github.com/celo-org/celo-monorepo/blob/d43293b159aee077989b4329e2396093ff874d4c/packages/protocol/test-sol/unit/common/CeloDistributionSchedule.t.sol#L69-L76